### ISSUE TITLE
`container` in `render` follow-up work

### ISSUE THREAD

**Issue by pvinis** [callstack/react-native-testing-library#669]
## Description

After `container` was exposed from `render` in 7.1.0, there is some extra work that needs to be done before it's easily usable by everyone (at least for people using TS).

- [ ] `typings/index.d.ts` is not updated to contain `container`.
- [ ] `ReactTestInstance` is not exported in `typings/index.d.ts` so I can't add this as a type to my helper test function
- [ ] I need access to `ErrorWithStack` and `getNodeByText`. Is there a way to use these on `container`?

**Comment by tulitheprogrammer** [comment]
It's so urgent... what is needed here? who can help us with that?

**Comment by thymikee** [comment]
@tulitheprogrammer you can help :) one little step at a time!

**Comment by mdjastrzebski** [comment]
Closing issue as stale, probably no longer relevant due to migration to TypeScript.

@pvinis please report a new GH issue if the problem is present in the current version of RNTL (v11) in the recommended setup as in the [basic example app](https://github.com/callstack/react-native-testing-library/tree/main/examples/basic). If so, please also provide a minimal repro repository.
